pkg/framework/coremodel/staticregistry/unfeatured.go

```diff
@@ -0,0 +1,86 @@
+package staticregistry
+
+import (
+	"github.com/grafana/grafana/pkg/cuectx"
+	"github.com/grafana/grafana/pkg/framework/coremodel"
+	"github.com/grafana/thema"
+)
+
+var emptyLin thema.Lineage
+
+const emptyLinRaw = `
+joinSchema: {...}
+name: "stub"
+seqs: [
+	{
+		schemas: [
+			{...}
+		]
+	}
+]
+`
+
+func init() {
```

Comment:
What is "unfeatured"?  Is it a schema that allows anything?

Reply:
yep, it's basically a stub that allows anything through. (any object, that is - it would fail to validate e.g. a string or number)

Reply:
Yep. Or at least, any object - doesn't allow primitives (string, list, array). It's basically just a runtime stub.